google-cloud-error_reporting/.rubocop.yml

```diff
@@ -3,6 +3,7 @@ AllCops:
     - "integration/**/*"
     - "google-cloud-error_reporting.gemspec"
     - "lib/google/devtools/**/*"
+    - "lib/google/cloud/error_reporting/**/*"
```

Comment:
True. This is something I'm not exactly sure. I asked @hxiong388 about this, but he seemed to be out of office. I will wait for his feedback.